readthedocs/analytics/models.py

```diff
@@ -86,50 +96,40 @@ def top_viewed_pages(
         :param per_version: If `True`, group the results per version.
 
         :returns: A list of named tuples ordered by the number of views.
-         Each tuple contains: path, full_path, and total_views.
+         Each tuple contains: path, url, and count.
         """
         if since is None:
             since = timezone.now().date() - timezone.timedelta(days=30)
 
-        group_by = ["path"]
-        values = ["path", "total_views"]
-        if per_version:
-            group_by.append("version")
-            values.append("version__slug")
-
+        group_by = "full_path" if per_version else "path"
         queryset = (
             cls.objects.filter(project=project, date__gte=since, status=status)
-            .values_list(*group_by)
-            .annotate(total_views=Sum("view_count"))
-            .values_list(*values, named=True)
-            .order_by("-total_views")[:limit]
+            .values_list(group_by)
+            .annotate(count=Sum("view_count"))
+            .values_list(group_by, "count", named=True)
+            .order_by("-count")[:limit]
         )
 
-        PageViewResult = namedtuple("PageViewResult", "path, full_path, count")
+        PageViewResult = namedtuple("PageViewResult", "path, url, count")
```

Comment:
I'm a little confused here between `url` and `full_path`. Are they different? I assume that `url` would include protocol, domain, etc and `full_path` just the URL's path. Am I correct?